docs/guide/interactions/modals.rst

```diff
@@ -0,0 +1,143 @@
+.. currentmodule:: discord
+
+.. _guide_interactions_modals:
+
+Modals
+========
+
+Modals are a form of interaction which prompt the user for additional information. They appear as a pop-up window and accept input using
+components such as :class:`~discord.ui.TextInput`.
+
+This section will detail how to create and use these modals in your code.
+
+
+Defining a Modal
+------------------
+
+Modals share structural similarities with :ref:`Views <guide_interactions_views>`, both are used to design component-based UIs.
+
+Modals are created by subclassing the :class:`~discord.ui.Modal` class.
+
+For example:
+
+.. code-block:: python
+
+    class FeedbackForm(discord.ui.Modal, title='Feedback'):
+        name = discord.ui.TextInput(label='Name')
+        feedback = discord.ui.TextInput(label='Feedback', style=discord.TextStyle.long)
+        additional_information = discord.ui.TextInput(
+            label='Additional Information', style=discord.TextStyle.long, required=False
+        )
+
+        async def on_submit(self, interaction: discord.Interaction) -> None:
+            ... # do something with the data
+
+            await interaction.response.send_message(f'Thank you {self.name.value}, your submission was recorded.')
+
+This produces a modal which appears on Discord as:
+
+.. image:: /images/guide/interactions/modal_example.png
+
+Let's break down the code:
+
+
+Title
```

Comment:
No brief example?

Reply:
I might have misread, and was referring to what a modal does. You can mark this resolved